### PR TITLE
[FLINK-6603] [streaming] Enable checkstyle on test sources

### PR DESCRIPTION
**Pull request by greghogan**
Follow-up to FLINK-6107.

The first commit sets the import order as [discussed](https://issues.apache.org/jira/browse/FLINK-6107?focusedCommentId=15985221&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15985221): first Flink imports, then other imports, then javax, then java (we could consider placing javax and java together; I have not looked into the required regex and we only have a few instances of javax). The groups are separated by a single newline with static imports moved to the bottom.

The second commit applies the strict checkstyle to the `flink-streaming-java` tests. The checkstyle line length was increased from 2500 to 3000.

### CODE REVIEW

File: flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java [hunk 1]
```diff
@@ -1237,9 +1240,9 @@ public void testProcessingTimeSessionWindows() throws Throwable {
 	}
 
 	@Test
-	public void testLateness() throws Exception {
-		final int WINDOW_SIZE = 2;
-		final long LATENESS = 500;
+	public void testlateness() throws Exception {
```
Comment (zentol) [inline flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java]: This change looks wrong, it should be "**L**ateness".

File: flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java [hunk 2]
```diff
@@ -1374,9 +1377,9 @@ public long getCurrentProcessingTime() {
 	}
 
 	@Test
-	public void testSideOutputDueToLatenessTumbling() throws Exception {
-		final int WINDOW_SIZE = 2;
-		final long LATENESS = 0;
+	public void testSideOutputDueTolatenessTumbling() throws Exception {
```
Comment (zentol) [inline flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java]: same here

File: flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java [hunk 3]
```diff
@@ -1438,10 +1441,10 @@ public void testSideOutputDueToLatenessTumbling() throws Exception {
 	}
 
 	@Test
-	public void testSideOutputDueToLatenessSliding() throws Exception {
-		final int WINDOW_SIZE = 3;
-		final int WINDOW_SLIDE = 1;
-		final long LATENESS = 0;
+	public void testSideOutputDueTolatenessSliding() throws Exception {
```
Comment (zentol) [inline flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java]: same here

File: flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java [hunk 4]
```diff
@@ -1518,9 +1521,9 @@ public void testSideOutputDueToLatenessSliding() throws Exception {
 	}
 
 	@Test
-	public void testSideOutputDueToLatenessSessionZeroLatenessPurgingTrigger() throws Exception {
-		final int GAP_SIZE = 3;
-		final long LATENESS = 0;
+	public void testSideOutputDueTolatenessSessionZerolatenessPurgingTrigger() throws Exception {
```
Comment (zentol) [inline flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java]: same here

File: flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java [hunk 5]
```diff
@@ -1610,9 +1613,9 @@ public void testSideOutputDueToLatenessSessionZeroLatenessPurgingTrigger() throw
 	}
 
 	@Test
-	public void testSideOutputDueToLatenessSessionZeroLateness() throws Exception {
-		final int GAP_SIZE = 3;
-		final long LATENESS = 0;
+	public void testSideOutputDueTolatenessSessionZerolateness() throws Exception {
```
Comment (zentol) [inline flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java]: same here

File: flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java [hunk 6]
```diff
@@ -1696,13 +1699,13 @@ public void testSideOutputDueToLatenessSessionZeroLateness() throws Exception {
 	}
 
 	@Test
-	public void testDropDueToLatenessSessionWithLatenessPurgingTrigger() throws Exception {
+	public void testDropDueTolatenessSessionWithlatenessPurgingTrigger() throws Exception {
```
Comment (zentol) [inline flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java]: .

File: flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java [hunk 7]
```diff
@@ -1783,13 +1786,13 @@ public void testDropDueToLatenessSessionWithLatenessPurgingTrigger() throws Exce
 	}
 
 	@Test
-	public void testNotSideOutputDueToLatenessSessionWithLateness() throws Exception {
-		// same as testSideOutputDueToLatenessSessionWithLateness() but with an accumulating trigger, i.e.
+	public void testNotSideOutputDueTolatenessSessionWithlateness() throws Exception {
```
Comment (zentol) [inline flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java]: .

File: flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java [hunk 8]
```diff
@@ -1887,10 +1890,10 @@ public void testNotSideOutputDueToLatenessSessionWithLateness() throws Exception
 	}
 
 	@Test
-	public void testNotSideOutputDueToLatenessSessionWithHugeLatenessPurgingTrigger() throws Exception {
+	public void testNotSideOutputDueTolatenessSessionWithHugelatenessPurgingTrigger() throws Exception {
```
Comment (zentol) [inline flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java]: .

File: flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java [hunk 9]
```diff
@@ -1979,9 +1982,9 @@ public void testNotSideOutputDueToLatenessSessionWithHugeLatenessPurgingTrigger(
 	}
 
 	@Test
-	public void testNotSideOutputDueToLatenessSessionWithHugeLateness() throws Exception {
-		final int GAP_SIZE = 3;
-		final long LATENESS = 10000;
+	public void testNotSideOutputDueTolatenessSessionWithHugelateness() throws Exception {
```
Comment (zentol) [inline flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperatorTest.java]: .

**Comment by greghogan** [comment]
Thanks for the review @zentol. The mistake was caused by not performing a case-sensitive find-replace. I redid the file with variable rename.

**Comment by zentol** [comment]
+1 to merge.

**Comment by greghogan** [comment]
@StephanEwen please verify the first commit implements the desired/traditional import order.

**Comment by StephanEwen** [comment]
@greghogan Awesome, looks good! Thanks a lot for taking this into account!

**Comment by zentol** [comment]
@greghogan Do we have a checkstyle rule for an empty line before the package declaration? I saw you added a commit for it, but couldn't find the corresponding checkstyle rule.

Travis is also failing at the moment due to a checkstyle violation:

```[ERROR] src/test/java/org/apache/flink/streaming/util/OperatorSnapshotUtil.java:[51] (regexp) RegexpSingleline: Trailing whitespace```

**Comment by greghogan** [comment]
@zentol added `EmptyLineSeparator` to the strict checkstyle.

**Comment by zentol** [comment]
@greghogan +1, feel free to merge it :)

**Pull request closed without merging**